src/Feature/Layout/Feature_Layout.re

```diff
@@ -1,5 +1,10 @@
 // MODEL
 
+type panel =
+  | Left
+  | Center(int)
+  | Bottom;
```

Comment:
This can easily be extended with `Top` and `Right` as well, but I didn't want to take that into account everywhere as long as we don't use it.